src/block/table-of-contents/edit.js

```diff
@@ -424,31 +426,41 @@ const Edit = props => {
 				</PanelAdvancedSettings>
 			</InspectorStyleControls>
 
-			<InspectorStyleControls>
-			</InspectorStyleControls>
-
 			<Typography.InspectorControls
+				label={ __( 'Table of Contents', i18n ) }
 				isMultiline={ true }
 				initialOpen={ false }
 				hasTextTag={ false }
 				hasTextContent={ false }
 			/>
 
+			<Typography.InspectorControls
+				label={ __( 'Title', i18n ) }
+				attrNameTemplate="title%s"
+				initialOpen={ false }
+				hasToggle={ true }
+				hasTextTag={ false }
+				hasTextContent={ false }
+			/>
+
 			<Advanced.InspectorControls />
 			<Transform.InspectorControls />
 			<EffectsAnimations.InspectorControls />
 			<CustomAttributes.InspectorControls />
 			<CustomCSS.InspectorControls mainBlockClass="stk-table-of-contents" />
 			<Responsive.InspectorControls />
 			<ConditionalDisplay.InspectorControls />
-
 			<TableOfContentsStyles version={ VERSION } />
 			<CustomCSS mainBlockClass="stk-table-of-contents" />
-
 			<BlockDiv className={ blockClassNames }>
 				{ !! headings.length && hasEmptyAnchor && (
 					<Notice autoGenerateAnchors={ autoGenerateAnchors } />
 				) }
+				{ attributes.titleShow && <Typography
+					className="stk-table-of-contents__title"
+					attrNameTemplate="title%s"
+					defaultValue="Table of Contents"
```

Comment:
If you delete the text, won't this auto-place "Table of Contents" in the field? Is it possible to remove the `defaultValue` here? I think the default value should be handled by the `default` value in the attribute schema.
```suggestion
```